andela/strikers-ah-backend
# Ch 164489777 integrating coveralls

#### What does this PR do?
  This PR integrates coveralls for code coverage
#### Description of Task to be completed?
  -Installing coveralls.
  -Installing mocha-lcov-reporter to report to coveralls
#### How should this be manually tested?
  -Clone the project
  -run npm i
  -when running tests you should be able to see coverage percentage
#### What are the relevant pivotal tracker stories?
  [Integrate Coveralls code coverage service #164489777](https://www.pivotaltracker.com/story/show/164489777)
